### PR TITLE
fix documentation for findMany JSON AnyNull

### LINKED ISSUE #3609: Incorrect documentation for JSON AnyNull
In
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-by-null-values

the example on how to query for isNull JSON fields is:

```
import { Prisma } from '@prisma/client'

prisma.log.findMany({
  where: {
    data: {
      meta: Prisma.AnyNull,
    },
  },
})
```

For me I get a: `Object literal may only specify known properties, and 'data' does not exist in type 'LogWhereInput'`

the correct way to do it seems to be:

```
import { Prisma } from '@prisma/client'

prisma.log.findMany({
  where: {
     meta: {
       equals: Prisma.AnyNull,
     },
  },
})
```
I think the documentation needs to be updated.

### PR DESCRIPTION
**Pull request by dadaphl**
## Describe this PR

PR fixes mistake in documentation on query filter for JSON AnyNull.

## Changes

Fixes example on how to query for AnyNull on JSON column.

## What issue does this fix?

Fixes #3609 

## Any other relevant information

N/A

### CODE REVIEW

**Comment by songkeys** [comment]
It took me half an hour to debug my code (checked version, regenerated types, searched on internet, tried minimal reproduction) and finally found it might be a doc typo. I was about to make this PR and found it has been made.

This is frustrating. Please merge this PR as others may encounter the same situation.

**Review by Frosty21 — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx [hunk 1]
```diff
@@ -885,9 +885,9 @@ import { Prisma } from '@prisma/client'
 
 prisma.log.findMany({
   where: {
-    data: {
-      meta: Prisma.AnyNull,
-    },
+     meta: {
+       equals: Prisma.AnyNull,
+     },
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx:890]: Fixing the indentation of the changed code snippet:
```suggestion
    meta: {
      equals: Prisma.AnyNull,
    },
```

Comment (Frosty21) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx:890]: my bad thought it was already being auto indented with prettier after husky pre-commit that's why i approved it...

**Comment by tanberry** [comment]
Thanks so much for catching and fixing this, @dadaphl ! Apologies it took a while to get it merged.

**Pull request merged**